### PR TITLE
Updated the shell script to prevent -m < 256M. Fixes #388

### LINKED ISSUE #388: bad error message when giving -m less than 256M
_Original author: jeff.al...@gmail.com (May 22, 2011 22:48:28)_

<b>What steps will reproduce the problem?</b>
1. ./refine -m 128M
2. Get error &quot;incompatible min and max heap size&quot; or something.

<b>What is the expected output? What do you see instead?</b>

I expected it to just work, even on my little machine. It didn't. I discovered the -m argument. I expected it to work, it didn't. I read the shell script and realized I had to hack it to lower the default 256M to 128M.

What version of Google Refine are you using?  r1836

What operating system and browser are you using? Linux, Firefox

<b>Is this problem specific to the type of browser you're using or it happens</b>
in all the browsers you tried? No

_Original issue: http://code.google.com/p/google-refine/issues/detail?id=388_

### PR DESCRIPTION
**Pull request by aakankasha028**

### CODE REVIEW

**Comment by wetneb** [comment]
Thank you! But you seem to have pulled master in your branch. Could you rebase instead, such that your pull request contains only your own commit? Let me know if you need help with that.

File: refine [hunk 1]
```diff
@@ -657,7 +658,7 @@ broker_appengine_run() {
 findbugs() {
     findbugs_prepare
     
-    MVN_PARAMS="-Dfindbugs.dir=${REFINE_TOOLS_DIR}/${FINDBUGS_DIR}"
+    MVN_PARAMS="-Dfindbugs.dir=${REFINE_TOOLS_DIR}/${FINDBUGS_DIR}"f
```
Comment (wetneb) [inline refine]: Is this "f" really meant to be there?

Comment (aakankasha028) [reply refine]: No no, that was by mistake. I'll amend that in the new PR

**Comment by wetneb** [comment]
Duplicate of #2490.

**Pull request closed without merging**